.gitignore

```diff
@@ -1,2 +1,4 @@
 /lex.yy.c
 /cvsclone
+
+*.dSYM
```

Comment:
Could you please explain in the commit message what the dSYM files are and/or where do they come from? (why they need to appear in the gitignore).